spec/outputs/loggly_spec.rb

```diff
@@ -1 +1,60 @@
+# encoding: utf-8
 require "logstash/devutils/rspec/spec_helper"
+require "logstash/outputs/loggly"
+
+describe 'outputs/loggly' do
+  let(:config) { { "key" => '34567d8-2b67-567-fghj-6789' } }
```

Comment:
Are API keys on Loggly not secret? Or is this just a fake key to satisfy some of the tests? 
If this is a real key, recommend you expire this key immediately!